tests/simpleline_tests/glib_tests/event_loop_glib_test.py

```diff
@@ -30,3 +30,7 @@ def tearDown(self):
 
     def create_loop(self):
         self.create_glib_loop()
+
+
+# Hack to avoid running the original class thanks to import
+del ProcessEvents_TestCase
```

Comment:
I think that this could be fixed with defined `__all__`.

Reply:
Sadly, nope it doesn't work.

Reply:
Ok, then this solution is probably the best.